src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -2300,8 +2300,13 @@ public String getContextPath() throws IOException {
          * Use {@link #createCrumbedUrl} instead if you intend to call {@link WebRequest#setRequestBody}, typical of a POST request.
```

Comment:
You should also fix `createCrumbedUrl` to look for `?` in `relativePath` and switch to `&`.

Reply:
ok, will take a look

Reply:
@jglick addressed [here](https://github.com/jenkinsci/jenkins-test-harness/pull/97/files#diff-1d6816bd6f9430280c5d8b7853286b1dR2320)